Specs/Core/loadKTX2Spec.js

```diff
@@ -405,247 +251,13 @@ describe("Core/loadKTX2", function () {
     0,
     255,
   ]);
-  var validUncompressedMipmap = new Uint8Array([
-    171,
-    75,
-    84,
-    88,
-    32,
-    49,
-    49,
-    187,
-    13,
-    10,
-    26,
-    10,
-    1,
-    2,
-    3,
-    4,
-    1,
-    20,
-    0,
-    0,
-    1,
-    0,
-    0,
-    0,
-    7,
-    25,
-    0,
-    0,
-    81,
-    128,
-    0,
-    0,
-    7,
-    25,
-    0,
-    0,
-    4,
-    0,
-    0,
-    0,
-    4,
-    0,
-    0,
-    0,
-    0,
-    0,
-    0,
-    0,
-    0,
-    0,
-    0,
-    0,
-    1,
-    0,
-    0,
-    0,
-    3,
-    0,
-    0,
-    0,
-    52,
-    0,
-    0,
-    0,
-    19,
-    0,
-    0,
-    0,
-    67,
-    82,
-    78,
-    76,
-    73,
-    66,
-    95,
-    70,
-    79,
-    85,
-    82,
-    67,
-    67,
-    0,
-    82,
-    71,
-    66,
-    120,
-    0,
-    0,
-    23,
-    0,
-    0,
-    0,
-    75,
-    84,
-    88,
-    111,
-    114,
-    105,
-    101,
-    110,
-    116,
-    97,
-    116,
-    105,
-    111,
-    110,
-    0,
-    83,
-    61,
-    114,
-    44,
-    84,
-    61,
-    100,
-    0,
-    0,
-    48,
-    0,
-    0,
-    0,
-    255,
-    0,
-    0,
-    255,
-    0,
-    0,
-    255,
-    0,
-    0,
-    255,
-    0,
-    0,
-    255,
-    0,
-    0,
-    255,
-    0,
-    0,
-    255,
-    0,
-    0,
-    255,
-    0,
-    0,
-    255,
-    0,
-    0,
-    255,
-    0,
-    0,
-    255,
-    0,
-    0,
-    255,
-    0,
-    0,
-    255,
-    0,
-    0,
-    255,
-    0,
-    0,
-    255,
-    0,
-    0,
-    255,
-    0,
-    0,
-    12,
-    0,
-    0,
-    0,
-    255,
-    0,
-    0,
-    255,
-    0,
-    0,
-    255,
-    0,
-    0,
-    255,
-    0,
-    0,
-    3,
-    0,
-    0,
-    0,
-    255,
-    0,
-    0,
-    0,
-  ]);
-
-  var fakeXHR;
-
-  beforeEach(function () {
-    fakeXHR = jasmine.createSpyObj("XMLHttpRequest", [
-      "send",
-      "open",
-      "setRequestHeader",
-      "abort",
-      "getAllResponseHeaders",
-    ]);
-    fakeXHR.simulateLoad = function (response) {
-      fakeXHR.simulateHttpResponse(200, response);
-    };
-    fakeXHR.simulateError = function () {
-      fakeXHR.response = "";
-      if (typeof fakeXHR.onerror === "function") {
-        fakeXHR.onerror();
-      }
-    };
-    fakeXHR.simulateHttpResponse = function (statusCode, response) {
-      fakeXHR.status = statusCode;
-      fakeXHR.response = response;
-      if (typeof fakeXHR.onload === "function") {
-        fakeXHR.onload();
-      }
-    };
-
-    spyOn(window, "XMLHttpRequest").and.returnValue(fakeXHR);
-  });
-
   it("throws with no url", function () {
     expect(function () {
       loadKTX2();
     }).toThrowDeveloperError();
   });
 
-  it("creates and sends request without any custom headers", function () {
```

Comment:
I couldn't find other examples where we have tests like this.. is it important to test url request behavior from `loadKTX2`?